package.json

```diff
@@ -107,7 +107,7 @@
     "!(*.d).ts": [
       "false -- Please use .tsx instead of .ts!"
     ],
-    "*/**/*.js": [
+    "src/**/*.js": [
```

Comment:
I don't think we've made any backend JS changes since introducing this pre-commit hook; it was objecting to the JS functions not being in Typescript. This changes it to only run that check for client app source files.